go.mod

```diff
@@ -18,19 +18,19 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
+	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
 	golang.org/x/tools v0.1.10
-	k8s.io/api v0.23.3
-	k8s.io/apiextensions-apiserver v0.23.3
-	k8s.io/apimachinery v0.23.3
-	k8s.io/autoscaler/vertical-pod-autoscaler v0.10.0
+	k8s.io/api v0.24.2
+	k8s.io/apiextensions-apiserver v0.24.2
+	k8s.io/apimachinery v0.24.2
+	k8s.io/autoscaler/vertical-pod-autoscaler v0.11.0
```

Comment:
Don't we need to update the replace directive for `k8s.io/autoscaler/vertical-pod-autoscaler` to point to v0.11.0 as well?

Reply:
Addressed in follow-up PR #570.